src/interface.jl

```diff
@@ -31,111 +46,201 @@ ADBackend(::Val) = error("The requested AD backend is not available. Make sure t
 ######################
 # Bijector interface #
 ######################
-"Abstract type for a bijector."
-abstract type AbstractBijector end
+"""
+
+Abstract type for a transformation.
+
+# Implementing
+
+A subtype of `Transform` of should at least implement [`transform(b, x)`](@ref).
```

Comment:
My main concern here is a bit that the names `Transform` and `transform` seem a bit generic, and if exported likely conflict with exports of other packages.

Reply:
True but it also seems like quite a natural naming :confused: